sys/linux/socket_netlink_generic_devlink.txt

```diff
@@ -92,3 +112,61 @@ devlink_trap_group_set_policy {
 	DEVLINK_ATTR_TRAP_GROUP_NAME	nlattr[DEVLINK_ATTR_TRAP_GROUP_NAME, string["l2_drops"]]
 	DEVLINK_ATTR_TRAP_ACTION	nlattr[DEVLINK_ATTR_TRAP_ACTION, int8[DEVLINK_TRAP_ACTION_DROP:DEVLINK_TRAP_ACTION_TRAP]]
 } [packed, align[4]]
+
+devlink_sb_get_policy {
+	handle			devlink_handle
+	DEVLINK_ATTR_SB_INDEX	nlattr[DEVLINK_ATTR_SB_INDEX, int32]
+} [packed, align[4]]
+
+devlink_sb_pool_get_policy {
+	handle				devlink_handle
+	DEVLINK_ATTR_SB_INDEX		nlattr[DEVLINK_ATTR_SB_INDEX, int32]
+	DEVLINK_ATTR_SB_POOL_INDEX	nlattr[DEVLINK_ATTR_SB_POOL_INDEX, int16]
+} [packed, align[4]]
+
+devlink_sb_pool_set_policy {
+	handle					devlink_handle
+	DEVLINK_ATTR_SB_INDEX			nlattr[DEVLINK_ATTR_SB_INDEX, int32]
+	DEVLINK_ATTR_SB_POOL_INDEX		nlattr[DEVLINK_ATTR_SB_POOL_INDEX, int16]
+	DEVLINK_ATTR_SB_POOL_SIZE		nlattr[DEVLINK_ATTR_SB_POOL_SIZE, int32]
+	DEVLINK_ATTR_SB_POOL_THRESHOLD_TYPE	nlattr[DEVLINK_ATTR_SB_POOL_THRESHOLD_TYPE, int8[DEVLINK_SB_THRESHOLD_TYPE_STATIC:DEVLINK_SB_THRESHOLD_TYPE_DYNAMIC]]
+} [packed, align[4]]
+
+devlink_sb_port_pool_get_policy {
+	port_handle			devlink_port_handle
+	DEVLINK_ATTR_SB_INDEX		nlattr[DEVLINK_ATTR_SB_INDEX, int32]
+	DEVLINK_ATTR_SB_POOL_INDEX	nlattr[DEVLINK_ATTR_SB_POOL_INDEX, int16]
+} [packed, align[4]]
+
+devlink_sb_port_pool_set_policy {
+	port_handle			devlink_port_handle
+	DEVLINK_ATTR_SB_INDEX		nlattr[DEVLINK_ATTR_SB_INDEX, int32]
+	DEVLINK_ATTR_SB_POOL_INDEX	nlattr[DEVLINK_ATTR_SB_POOL_INDEX, int16]
+	DEVLINK_ATTR_SB_THRESHOLD	nlattr[DEVLINK_ATTR_SB_THRESHOLD, int32]
+} [packed, align[4]]
+
+devlink_sb_tc_pool_bind_get_policy {
+	port_handle			devlink_port_handle
+	DEVLINK_ATTR_SB_INDEX		nlattr[DEVLINK_ATTR_SB_INDEX, int32]
+	DEVLINK_ATTR_SB_TC_INDEX	nlattr[DEVLINK_ATTR_SB_TC_INDEX, int16]
+	DEVLINK_ATTR_SB_POOL_TYPE	nlattr[DEVLINK_ATTR_SB_POOL_TYPE, int8[DEVLINK_SB_POOL_TYPE_INGRESS:DEVLINK_SB_POOL_TYPE_EGRESS]]
+} [packed, align[4]]
+
+devlink_sb_tc_pool_bind_set_policy {
```

Comment:
Since we use a struct here, syzkaller will not vary set of attributes in this message. It will always generate these 6, no more and no less. Is it what we want? We generally use an array of unions with all attributes so that fuzzer can generate all possible combinations.